tests/python/relay/test_op_qnn_dense.py

```diff
@@ -75,52 +75,8 @@ def make_configuration(quantized_data,
     return config
 
 
-def make_uint_configuration(use_bias=False, requantize_output=False):
```

Comment:
This is removed because it is not used in the tests and I get segfault from LLVM when I tried to use it.

Reply:
Yes, we looked into that concluded it is an LLVM bug